<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindowc</class>
 <widget class="QDialog" name="mainwindowc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>610</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#pushButton{
	font: 18pt &quot;Bahnschrift&quot;;

	background-color: rgb(179, 179, 193);
	
	
	
	color: rgb(0, 0, 127);
	
;}


#pushButton:hover{
;

	color: rgb(255, 255, 255);
	background-color: rgb(4, 44, 89);
	}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_password">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>510</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#LE_password{

background-color: rgb(255, 255, 255);
	
	color: rgb(4, 44, 89);
	
;}


#LE_password:hover{
;

	color: rgb(255, 255, 255);
	background-color: rgb(4, 44, 89);
	}</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-610</x>
     <y>-200</y>
     <width>1331</width>
     <height>701</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>:/new/prefix1/Objet-dynamique-vectoriel.png</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="LE_username">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#LE_username{

background-color: rgb(255, 255, 255);
	
	color: rgb(4, 44, 89);
	
;}


#LE_username:hover{
;

	color: rgb(255, 255, 255);
	background-color: rgb(4, 44, 89);
	}</string>
   </property>
  </widget>
  <widget class="QLabel" name="l_titre">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>1191</width>
     <height>721</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/interface/Untitled-zrf.png);</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>LE_username</zorder>
  <zorder>l_titre</zorder>
  <zorder>LE_password</zorder>
  <zorder>pushButton</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
